monitor.go

```diff
@@ -576,29 +644,18 @@ func runMonitor() {
 			if !pushTestImage(dockerClient) {
 				duration = 30 * time.Second
 				// Write the failure metric.
-				m, err := promFailureMetric.GetMetricWithLabelValues()
-				if err != nil {
-					panic(err)
-				}
-
-				m.Inc()
-
+				reportFailure(cloudwatchService)
 				continue
 			}
 
 			// Write the push time metric.
-			promPushMetric.Observe(time.Since(pushStartTime).Seconds())
+			reportPushTime(cloudwatchService, time.Since(pushStartTime))
 
 			log.Infof("Test successful")
 			duration = userDuration
 
 			// Write the success metric.
-			m, err := promSuccessMetric.GetMetricWithLabelValues()
```

Comment:
I feel like we should support both prometheus and cloudwatch, because there are probably users of this tool using prometheus, and we'll eventually switch back to it when we move this service

Reply:
This should be supporting both now